tests/main/nfs-support/task.yaml

```diff
@@ -0,0 +1,136 @@
+summary: Test that snaps still work when /home is a NFS mount
+details: |
+    Snapd now contains a feature where NFS-mounted /home (or any sub-directory)
+    initializes a workaround mode where all snaps gain minimal amount of network
+    permissions sufficient for NFS to operate.
+systems: [ubuntu-16.04-64]  # TODO: expand this list
+prepare: |
+    . "$TESTSLIB/pkgdb.sh"
+    . "$TESTSLIB/snaps.sh"
+
+    # Install NFS server and a simple shell snap.
+    distro_install_package nfs-kernel-server
+    distro_install_package bsdgames
+    install_local test-snapd-sh
+execute: |
+    run_export_nfs() {
+        # NOTE: This is so silly because spellchecker complains about the real
+        # command name so we have to go and obfuscate it /o\
```

Comment:
Ouch. If we end up with more of those, we may as well drop spellchecker.

Reply:
It would be good to be able to teach the spellchecker about new words. I'll see if I can do that instead of this hack.

Reply:
Ha, that was actually easier than I thought. Done now.